codalib/util.py

```diff
@@ -11,8 +11,7 @@
 from lxml import etree
 import uuid
 import datetime
-from coda import bagatom
-import httplib2
```

Comment:
How do you feel about removing all of those commented out lines that refer to the the `httplib` object?